lib/Controller.php

```diff
@@ -178,7 +178,7 @@ private function _init()
         // force default language, if language selection is disabled and a default is set
         if (!$this->_conf->getKey('languageselection') && strlen($lang) == 2) {
             $_COOKIE['lang'] = $lang;
-            setcookie('lang', $lang, 0, '', '', true);
+            setcookie('lang', $lang, array('SameSite' => 'Strict', 'Secure' => true));
```

Comment:
Because cookies could get set from either end: In the frontend by user action if the user selects a value and in the backend "if language selection is disabled and a default is set" in the configuration file. One of the 4 cases as per matrix in https://github.com/PrivateBin/PrivateBin/wiki/Configuration#languageselection--languagedefault

Reply:
Still, cookie must not be ser ti Strict…

Reply:
My Firefox complains at me (just open your F12-console and visit any PB instance) because we don't do it, and the spec strongly encourages you do set it explicitly.

Usually you are the one arguing in favor of security related extensions? ;-)